samples/README.md

```diff
@@ -0,0 +1,5 @@
+# Samples for python-translate
```

Comment:
Would it be better to move this down to `samples/snippets/` instead of `samples/`?